guides/players/radios.md

```diff
@@ -84,7 +84,7 @@ To change the 148's channel you left click the Channel Selector to cycle up to t
 <hr/>
 
 # AN/PRC-117F, AN/VRC-103
-The AN/PRC-117 (or its vehicle configuration AN/VRC-103) is a long range command level radio carried in a backpack (or pre-mounted in vehicles) used by Squad Leaders, Command and Support elements to communicate with one another. 
+The AN/PRC-117 (or its vehicle configuration AN/VRC-103) is a long range command radio carried in a backpack (or pre-mounted in vehicles) used by Squad Leaders, Command and Support elements to communicate with one another. 

```

Comment:
Should be either `inter-squad` or `command level`

Reply:
I think its better to leave it as command, I removed level because it sounded nicer imho, just me though